lib/dialects/abstract/query-generator.js

```diff
@@ -919,7 +922,7 @@ class QueryGenerator {
    *
    * @returns {string}
    */
-  quoteIdentifier(identifier, force) {
+  quoteIdentifier(identifier, force) {	  
```

Comment:
@AWADHAMBIKA Remove space at end of line. undo all such unwanted changes in all files.